connection/connection.go

```diff
@@ -146,6 +147,7 @@ type options struct {
 
 // connect is the internal implementation of Connect. It has more options to enable testing.
 func connect(
+	ctx context.Context,
```

Comment:
Now that `connect` has a context, `gprc.Dial` should get replaced with `grpc.DialContext`.

Reply:
Resolved.